### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by Wesitos**
+ Fix badges
+ Fix code snippets

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,10 +162,10 @@ Here is a typical example:
     haraka
     qpsmtpd
     spamassassin
-
+```
 That produces the following Javascript object:
 
-````javascript
+```javascript
```
Comment (msimerson) [inline README.md]: You used ```js above, let's be consistent and use that here too.

File: README.md [hunk 2]
```diff
@@ -193,38 +193,38 @@ The key=value pairs support continuation lines using the
 backslash "\" character.
 
 The `options` object allows you to specify which keys are boolean:
-
+```javascript
```
Comment (msimerson) [inline README.md]: and here.

File: README.md [hunk 3]
```diff
@@ -193,38 +193,38 @@ The key=value pairs support continuation lines using the
...
+```
 For completeness the inverse is also allowed:
-
+```js
     { booleans: [ '-reject' ] }
-
+```
 Lists are supported using this syntax:
-
+```javascript
```
Comment (msimerson) [inline README.md]: and here.

File: README.md [hunk 4]
```diff
@@ -150,7 +150,7 @@ INI files have their heritage in early versions of Microsoft Windows.
 Entries are a simple format of key=value pairs, with optional [sections].
 
 Here is a typical example:
-
+```
```
Comment (msimerson) [inline README.md]: why not use ```ini here?

File: README.md (hunk 3 above)
Comment (msimerson) [inline README.md]: actually, it should be ```ini here.

File: README.md (hunk 4 above)
Comment (Wesitos) [inline README.md]: I didn't noticed that "ini" was recognized by github.

File: README.md (hunk 1 above)
Comment (Wesitos) [inline README.md]: Oh, I though that I had done that.

**Comment by Wesitos** [comment]
Done! I think I sent the pull request too soon :b

**Comment by msimerson** [comment]
thanks!

**Pull request merged**